Detectors/ITSMFT/ITS/tracking/include/ITStracking/Configuration.h

```diff
@@ -131,9 +131,20 @@ struct VertexerStoreConfigurationGPU {
   int processedTrackletsCapacity = maxTrackletsPerCluster * clustersPerLayerCapacity;
   int maxTrackletCapacity = 2e4;
   int maxCentroidsXYCapacity = std::ceil(maxTrackletCapacity * (maxTrackletCapacity - 1) / 2);
+  int nMaxVertices = 10;
   int nBinsXYZ[3] = {402, 402, 2002};
```

Comment:
You have here an almost identical definition of histos binning, i.e. ``struct VertexerHistogramsConfiguration`` repeats itself in the ``struct VertexerStoreConfigurationGPU`` (except the values of binSpanXYZ, is this on purpose?).
Isn't it safer to have a single definition,  in case something should be modified in future?

Also (for my own understanding): you define ``nBinsXYZ`` to some round value + offset then always use it decrememented by 1, e.g. ``nBinsXYZ[2] = 2002`` but the always ``nBinsXYZ[2]-1`` is used (did you really want 2001 bins in ``-40<z<40``?), why is this so?

Reply:
Agree with the comment on config, I just duplicated it temporary to make it easier to possibly remove everything in case we decide. Also while I was developing I just preferred to have nothing to affect also other piece of the code, to minimize the potential issues. Eventually I am indeed finding a unified configuration.

For the offsets I almost always use them decreased by one apart from some  `cub` routines which require it increased by one wrt you usually mean. Point is that those were the first to be configured so I just made it like that. Indeed also here we can just increase it in the cub calls.

Reply:
And with regard to the binning in z this is something to be adjusted, I will empirically find the best config, I will also probably bug you with a discussion on this at some point. But still, take it as placeholders of the final config.